src/styles/css/controls.css

```diff
@@ -0,0 +1,65 @@
+.controls {
+  color: #fff;
+  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
+  font-size: 16px;
+  position: fixed;
+  right: 0;
+  top: 0;
+  width: 100%; }
+  @media (max-width: 488px) {
+    .controls {
+      height: 4.65rem; } }
+  @media (min-width: 489px) {
+    .controls {
+      height: 2.5rem; } }
+  .controls .links {
+    background-color: #54514a;
+    height: inherit;
+    list-style-type: none;
+    margin: 0;
+    position: relative;
+    padding: 0.125rem;
+    text-align: center; }
+  .controls ul li {
+    float: left;
+    margin: 0.625rem; }
+  .controls a {
+    color: #fff; }
+
+.loading {
+  position: fixed;
+  width: 12.5rem;
+  height: 12.5rem;
+  text-align: center;
+  left: calc(50% - 6.25rem);
+  top: calc(50% - 6.25rem);
+  background-color: #76777b; }
+
+.controls-view {
+  position: fixed;
+  top: 0.125rem;
```

Comment:
This is making the settings view cover the nav, so you can't click it again to close the settings, which right now is the only way to close the settings.